### PR TITLE
en_bu OK

### PR DESCRIPTION
**Pull request by kyleburnette**

### CODE REVIEW

File: src/overlays/actors/ovl_En_Bu/z_en_bu.h [hunk 1]
```diff
@@ -5,9 +5,13 @@
...
 struct EnBu;
 
+typedef void (*EnBuActionFunc)(struct EnBu*, GlobalContext*);
+
 typedef struct EnBu {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x4C];
+    /* 0x144 */ EnBuActionFunc actionFunc;
+    /* 0x148 */ UNK_TYPE1 unk148[0x44];
+    /* 0x18C */ s32 unk18C;
```
Comment (isghj5) [inline src/overlays/actors/ovl_En_Bu/z_en_bu.h:14]: Suspicious? 0x44 pad followed by a display list ptr, but that ptr isn't assigned by anything?

Could this be some larger struct that also contains a pointer?

(Not that I'm suggesting you change anything, I just thought it was interesting)

Comment (Kenix3) [reply src/overlays/actors/ovl_En_Bu/z_en_bu.h:14]: Indeed, this should be a display list pointer rather than s32.

**Review by Kenix3 — CHANGES_REQUESTED** [review]

**Review by jpburnett — APPROVED** [review]
looks "bu"-tiful, I approve

**Review by Kenix3 — CHANGES_REQUESTED** [review]
Looks good to me.

**Review by Kenix3 — APPROVED** [review]
Meant to hit approve. Looks good!

**Pull request merged**